Automattic/jetpack
# Jetpack: create and update icons component of the VideoPress block

This PR creates a new icons component for the VideoPress block in order to make it independent of the Jetpack plugin.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: create and update icons component of the VideoPress block

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to block editor
* Take a look at the VideoPress block instance
* Confirm the icon is still there

<img width="667" alt="Screen Shot 2022-07-06 at 11 34 18 AM" src="https://user-images.githubusercontent.com/77539/177575800-3c876248-5575-4d42-bbeb-8d105e58907b.png">




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202514379845502
